package.json

```diff
@@ -26,11 +26,11 @@
     "test": "grunt test"
   },
   "devDependencies": {
-    "@dojo/cli-create-app": "^0.1.0",
-    "@dojo/interfaces": "~0.1.0",
+    "@dojo/cli": "file:./tarballs/dojo-cli-0.1.1-beta.1.tgz",
```

Comment:
Why do we use local paths for Dojo 2-related packages now?

Reply:
We don't. This is to get it to build for CI until the other packages are released.

Reply:
@devpaul Kit wrote this up last week: https://github.com/dojo/meta/wiki/Workflow-for-custom-package-releases